lib/Data/Validate/Sanctions/Redis.pm

```diff
@@ -50,12 +53,13 @@ sub get_sanctioned_info {
 sub _load_data {
     my $self = shift;
 
-    $self->{last_time}               //= 0;
+    $self->{last_modification}       //= 0;
+    $self->{last_index}              //= 0;
     $self->{_data}                   //= {};
     $self->{_sanctioned_name_tokens} //= {};
     $self->{_token_sanctioned_names} //= {};
 
-    my $last_time = $self->{last_time};
+    my $last_modification = $self->{last_modification};
```

Comment:
same here let's check when the last time we checked for an update if it's less than 5-10 min. no point in poking Redis, in the current implementation we'll fetch from Redis whore sanction list every dozen ms